core/src/main/kotlin/com/justai/jaicf/reactions/Reaction.kt

```diff
@@ -0,0 +1,85 @@
+package com.justai.jaicf.reactions
+
+import com.justai.jaicf.context.LoggingContext
+
+/**
+ * Abstraction for result of performing some reaction.
+ *
+ * @see Reactions
+ * */
+interface Reaction
+
+/**
+ * Result of performing reactions.say()
+ * */
+data class SayReaction internal constructor(
+    var text: String,
```

Comment:
I suppose you leave `var` here for the obfuscation purposes? In this case I'd investigate the possibility to make it immutable. For example via a separate visitor-like method `obfuscate` in the `Reaction` that receives an obfuscator and creates new obfuscated version of this one.